tasks/task724_DSTC2_classification.json

```diff
@@ -0,0 +1,1226 @@
+{
+    "Contributors": [
+        "Nisarg Patel"
+    ],
+    "Source": [
+        "https://github.com/matthen/dstc"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Definition": "Given the reponses by restaurant system, determine the experience of the customer understanding as strongly agree, agree,slightly agree,slightly disagree and strongly disagree.",
```

Comment:
Please correct the typo of the `responses`, consider adding a space after commas, and elaborate more on the definition as explained in the previous task.
I suggest merging strongly agree, agree, and slightly agree into agree and the others into disagree because the border between them might be fuzzy. Hence, average humans might not be able to clearly distinguish them.